ui/app/src/Views/SilenceForm/Types.elm

```diff
@@ -124,25 +136,50 @@ toSilence { id, comment, matchers, createdBy, startsAt, endsAt } =
 
 fromSilence : GettableSilence -> SilenceForm
 fromSilence { id, createdBy, comment, startsAt, endsAt, matchers } =
+    let
+        startsPosix =
+            case Utils.Date.timeFromString (DateTime.toString startsAt) of
```

Comment:
You can use [`Result.toMaybe`](https://package.elm-lang.org/packages/elm/core/latest/Result#toMaybe)